assets/javascript/index.js

```diff
@@ -0,0 +1,42 @@
+var topics = ['Impreza', 'WRX', 'Subaru', 'WRC', 'drift']
+topics.forEach(topic => {
+  $('#topicDiv').append(`
+    <button class="btn topicBtn" data-name="${topic}"> ${topic}</button>
+  `);
+});
+
+//user input button add
+$('#userAdd').on('click', function() {
+  event.preventDefault();
+  var topic = $('#userInput').val().trim();
+  if (topic === '') {
+  } else {$('#topicDiv').append(`<button class="btn topicBtn" data-name="${topic}">${topic}</button>`);
+} 
+$('#userInput').val('');
+})
+
+//giphy API pull
+$(document).on('click', '.topicBtn', function () {
```

Comment:
maybe target something more specific than the document, like the button container.